### PR TITLE
Provide information on MobilePay

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,26 +19,24 @@ END_METADATA -->
 
 <!-- END_COMMENT -->
 
-Here, you will find information related to integrating with the Vipps MobilePay APIs.
-
-Start your journey in the [Getting started](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/vipps-getting-started) guide.
-This leads you through applying for Vipps services and getting set up in the test environment.
+Vipps and MobilePay have recently merged, so the information is still under development.
```
Comment (cloveras) [inline README.md:22]:
```suggestion
Vipps and MobilePay have recently merged, so the information and developer documentation on this website is still under development. We are constantly making improvements.
```

File: contact.md [hunk 2]
```diff
@@ -131,19 +140,19 @@ Please send separate emails for each problem. If you send an email about a
 new problem by replying to an old issue, it may delay our response.
 If you reply to an old issue that has been closes, we will not see the reply.
 
-## Can I call you?
+### Can I call you?
 
-Nope. We do not have a phone number, and we do not have the capacity to
```
Comment (cloveras) [inline contact.md:136]: I liked "nope"! 😅

File: test-environment.md [hunk 3]
```diff
@@ -1,73 +1,59 @@
...
 
 <!-- END_COMMENT -->
 
-## About the test environment
+## Testing with MobilePay
+
+Vipps and MobilePay have recently merged so the information is still under development and will be updated soon.
+
+You can find MobilePay testing information under the _Development Guide > Testing_ section for each API in the
+[MobilePay developers website](https://developer.mobilepay.dk/).
```
Comment (cloveras) [inline test-environment.md:25]: Maybe skip the s?
```suggestion
[MobilePay developer website](https://developer.mobilepay.dk/).
```

**Review by cloveras — APPROVED** [review]
Nice! Pretty much good to go!

File: contact.md (hunk 2 above)
Comment (rebekaburnett) [reply contact.md:136]: I know, but it sounds like we are insensitive. :-D

File: test-environment.md (hunk 3 above)
Comment (rebekaburnett) [reply test-environment.md:25]: I'm going to revert changes to this whole page, based on Henriette's comment

**Pull request merged**